packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -224,6 +225,8 @@ class CupertinoSearchTextField extends StatefulWidget {
   // TODO(DanielEdrisian): Must make border radius continuous, see
   // https://github.com/flutter/flutter/issues/13914.
   final BorderRadius? borderRadius;
+  /// Set keyboard type for the input field. Default to TextInputType.text.
```

Comment:
```suggestion

  /// The keyboard type for this search field.
  ///
  /// Defaults to TextInputType.text.
```

Space is needed between properties and API docs.

Here we could just make reference to the existing property's API docs in `CupertinoTextField`.